airflow/decorators/base.py

```diff
@@ -101,27 +102,29 @@ class DecoratedOperator(BaseOperator):
     :type kwargs_to_upstream: dict
     """
 
-    template_fields: Iterable[str] = ('op_args', 'op_kwargs')
+    template_fields: Sequence[str] = ('op_args', 'op_kwargs')
     template_fields_renderers = {"op_args": "py", "op_kwargs": "py"}
 
     # since we won't mutate the arguments, we should just do the shallow copy
     # there are some cases we can't deepcopy the objects (e.g protobuf).
-    shallow_copy_attrs = ('python_callable',)
+    shallow_copy_attrs: Sequence[str] = ('python_callable',)
 
     def __init__(
         self,
         *,
         python_callable: Callable,
         task_id: str,
-        op_args: Tuple[Any],
-        op_kwargs: Dict[str, Any],
+        op_args: Optional[List] = None,
+        op_kwargs: Optional[Dict] = None,
```

Comment:
```suggestion
        op_kwargs: Optional[Dict[str, Any]] = None,
```

Reply:
Sad panda. I know better. I'll make sure PythonOperator and the like aligns.